.github/stale.yml

```diff
@@ -0,0 +1,63 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 7
```

Comment:
Since the bot is not sophisticated enough to have different `daysUntilStale` configs, we can't make it behave as in the [diagram](https://github.com/emqx/emqx/blob/master/.github/ISSUE_TEMPLATE/assets/issue-handling.png) for the `needs_information` state, since it is 14 days.

I guess it'd be OK for us to use 7 days uniformly?  If so, I can update the diagram for that transition and change it to 7 days as well.